monai/networks/schedulers/ddpm.py

```
@@ -125,7 +125,7 @@ def set_timesteps(self, num_inference_steps: int, device: str | torch.device | N
         step_ratio = self.num_train_timesteps // self.num_inference_steps
```

Ruff (0.14.3)

Local variable `step_ratio` is assigned to but never used

Remove assignment to unused variable `step_ratio`

(F841)